content/blog/posts/o3de-ux-past-present-future.md

```diff
@@ -0,0 +1,107 @@
+​
+>"_It means being brave enough to fail if it means that by applying lessons learnt, we can better surprise and delight customers in the future._"&mdash;[**Daniel Slater**](https://aws.amazon.com/executive-insights/content/how-amazon-defines-and-operationalizes-a-day-1-culture/)
+​
+A user's first day typically consists of the Installation and Project Setup, or the user onboarding experience. User onboarding primarily involves users manoeuvring through a product so they can achieve results faster while trying to understand the product's value. In this case, the game engine.
+​
+To learn more about our users' understanding of their Day 1 experience, we carried out numerous studies by interviewing, surveying, and testing over 40 users with varied backgrounds. As a result of our research, we were able to identify the typical behavioural patterns of our users. This allowed us to propose areas of improvement&mdash;the Project Manager and Gem Catalog.
+​
+At this point, I worked with the UX team to redesign the Project Manager and Gem Catalog. For those who are new to this, Gems are packages of code or assets that augment game or simulation projects, and the Gem Catalog is a library of Gems that are native to O3DE. We conducted user interviews to understand the overall experience of the Gem system, any suggestions and issues, and the frequency with which users configure their project with Gems.
+​
+>"_One of our biggest problems is not knowing what Gems are given to us, when installing or upgrading the application._"&mdash;**O3DE Research Participant**
+​
+We found that it was difficult for new users to wrap their mind around the concept of Gems and their use within projects. With 80 Gems native to O3DE, the typical usage varies over two patterns—users either retain the default configuration throughout their development process or they enable all Gems to shorten build and compilation time. It was important to establish an information hierarchy, because of the Gems' low ==findability== and ==discoverability==.
+​
+Findability
+: The ease with which information can be found within a system.
+​
+Discoverability
+: The degree to which information can be found within a system.
+​
+The fundamental objective for the system's architecture was to allow users to be aware of where they are, what they found, what is around, and what to expect. As a result, we conducted a card sort study where participants were asked to arrange a stack of virtual cards into distinct categories based on their understanding of Gems. We identified the key category for the faceted navigation to be *Features*, which would classify Gems on the basis of their distinctive attributes and functionalities.
```

Comment:
I dont have any word suggestions for now, but the formatting of this whole section seems incorrect. 

I've attached a screenshot of how the formatting of this paragraph is supposed to look like for this section. If we could replicate that please.

![its always day 1](https://user-images.githubusercontent.com/82846749/150595892-d16d24b6-2a83-461a-85e1-513d5de290c3.png)